### PR TITLE
fix(language): remove catalan opening punctuation (@Perseus333)

### PR DESCRIPTION
**Pull request by Perseus333**
### Description

Previously, Spanish exclusive punctuation was included in the Catalan language. In Catalan "¿" and "¡" don't exist. I just made it so that it adds the "¿" and "¡" if the current language is set to spanish.

### Checks

- [ ] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [ x Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username prefixed with @ inside parentheses at the end of the PR title.

Closes #

### CODE REVIEW

**Review by Miodec — APPROVED** [review]
Thanks

File: frontend/src/ts/test/words-generator.ts [hunk 1]
```diff
@@ -57,10 +57,14 @@ export async function punctuateWord(
     if (currentLanguage === "spanish" || currentLanguage === "catalan") {
```
Comment (Miodec) [inline frontend/src/ts/test/words-generator.ts:57]: Wait, so, if you check if the language is spanish in both cases inside this if, why not just remove this or condition?

Comment (Perseus333) [reply frontend/src/ts/test/words-generator.ts:57]: How would you remove the or condition? You still need the "!" and "?" in Catalan. How would you optimize this? If you removed the or, then you would need to have a separate if statement with duplicate logic. 

Or maybe there is something evident that I'm not seeing (very possible). Let me know.

Comment (Miodec) [reply frontend/src/ts/test/words-generator.ts:57]: You can just remove the `currentLanguage === "catalan"` check on that line and one line `74`, the rest is already handled.

Comment (Perseus333) [reply frontend/src/ts/test/words-generator.ts]: Now it should be fixed. Thanks for the help, @Miodec!

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**